### PR TITLE
UIEH-1005: Provider Record: List of Packages Results | Apply Next/Previous Component

### PR DESCRIPTION
**Pull request by mariiaschv**
## Purpose
Apply Next/Previous Component for loading provider packages

## Screenshots
tGgD2Wfb2A

## Issue
[UIEH-1005](https://issues.folio.org/browse/UIEH-1005)

### CODE REVIEW

File: src/components/provider/show/provider-show.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ class ProviderShow extends Component {
     listType: PropTypes.string.isRequired,
     model: PropTypes.object.isRequired,
     onEdit: PropTypes.func.isRequired,
-    packages: PropTypes.object.isRequired,
+    providerPackages: PropTypes.object.isRequired,
```
Comment (Vladyslav-Velytskyi) [inline src/components/provider/show/provider-show.js:46]: Please, describe the object

File: src/redux/reducers/providerPackages.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    hasLoaded: false,
+    hasFailed: false,
+    errors: [],
+  }),
+  [GET_PROVIDER_PACKAGES_SUCCESS]: (state, { payload }) => ({
+    ...state,
+    isLoading: false,
+    hasLoaded: true,
+    hasFailed: false,
+    items: [...payload.data],
```
Comment (Vladyslav-Velytskyi) [inline src/redux/reducers/providerPackages.js:34]:
```suggestion
    items: payload.data,
```

File: src/redux/reducers/tests/providerPackages.test.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        data: ['item1', 'item2'],
+        totalResults: 2,
+      },
+    };
+
+    expect(providerPackagesReducer(state, action)).toEqual({
+      ...state,
+      hasLoaded: true,
+      totalResults: 2,
+      items: [...action.payload.data],
```
Comment (Vladyslav-Velytskyi) [inline src/redux/reducers/tests/providerPackages.test.js:46]:
```suggestion
      items: action.payload.data,
```

File: src/routes/provider-show.js [hunk 4]
```diff
@@ -33,8 +42,11 @@ class ProviderShowRoute extends Component {
     location: ReactRouterPropTypes.location.isRequired,
     match: ReactRouterPropTypes.match.isRequired,
     model: PropTypes.object.isRequired,
+    providerPackages: PropTypes.shape({
+      items: PropTypes.array.isRequired,
```
Comment (Vladyslav-Velytskyi) [inline src/routes/provider-show.js:46]: Please create a shape for item prop type

**Comment by Vladyslav-Velytskyi** [comment]
Could you also add tests for `ProviderShowRoute`?

**Review by BogdanDenis — APPROVED** [review]

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Pull request merged**